connectors/aliyun/rocketmq-connect-http/README.md

```diff
@@ -0,0 +1,37 @@
+# rocketmq-connect-http
+
+## rocketmq-connect-http 打包
+```
+mvn clean install -Dmaven.test.skip=true
+```
+
+## rocketmq-connect-http 启动
+
+* **http-sink-connector** 启动
```

Comment:
How do I verify that this connector has been successfully started and is working well?